nav2_util/include/nav2_util/simple_action_server.hpp

```diff
@@ -193,41 +197,45 @@ class SimpleActionServer
     return current_handle_->is_canceling();
   }
 
-  void cancel_all(
+  // <PR FEEDBACK REQUEST> I'm trying to simplify even further how the user of the action server
```

Comment:
You should put PR comments in the PR instead of in code comments. You should probably remove this or turn it into a code comment explaining the purpose of the terminate goals function.